doc/release-notes/5.7-release-notes.md

```diff
@@ -0,0 +1,109 @@
+- Specification of controlled vocabulary terms in duplicate elements in multiple languages (in the installation default langauge and, if different, the dataset metadata language)
+
+While these changes are intended to improve harvesting and integration with external systems, they could break existing connections that make assumptions about the elements and attributes that have been changed.
+
+## New JVM Options and DB Settings
+
+- :SendNotificationOnDatasetCreation - A boolean setting that, if true will send an email and notification to users when a Dataset is created. Messages go to those, other than the dataset creator, who have the ability/permission necessary to publish the dataset.
+- :DatasetChecksumValidationSizeLimit - disables the checksum validation while publishing for any dataset size greater than the limit.
+- :DataFileChecksumValidationSizeLimit - Disables the checksum validation while publishing for any datafiles greater than the limit.
+- :CVocConf - A JSON-structured setting that configures Dataverse to associate specific metadatablock fields with external vocabulary services and specific vocabularies/sub-vocabularies managed by that service.
+- :MetadataLanguages - Sets which languages can be used when entering dataset metadata.
+
+## Notes for Tool Developers and Integrators
+
+### Bags Now Support File Paths
+
+The original Bag generation code stored all dataset files directly under the /data directory. With the addition in Dataverse of a directory path for files and then a change to allow files with different paths to have the same name, archival Bags will now use the directory path from Dataverse to avoid name collisions within the /data directory. Prior to this update, Bags from Datasets with multiple files with the same name would have been created with only one of the files with that name (with warnings in the log, but still generating the Bag).
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.7 Milestone](https://github.com/IQSS/dataverse/milestone/98?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Community Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.7/installation/).
+
+## Upgrade Instructions
+
+0\. These instructions assume that you've already successfully upgraded from Dataverse Software 4.x to Dataverse Software 5 following the instructions in the [Dataverse Software 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0). After upgrading from the 4.x series to 5.0, you should progress through the other 5.x releases before attempting the upgrade to 5.7.
+
```

Comment:
We keep losing the part of this section 0. that were added recently, to remind people to run these commands as the proper OS user, etc. The lines in question are below (as seen in the 5.6 release instruction). Do we have a release template somewhere that needs to be updated?

```
If you are running Payara as a non-root user (and you should be!), **remember not to execute the commands below as root**. Use `sudo` to change to that user first. For example, `sudo -i -u dataverse` if `dataverse` is your dedicated application user.  

In the following commands we assume that Payara 5 is installed in `/usr/local/payara5`. If not, adjust as needed.

`export PAYARA=/usr/local/payara5`

(or `setenv PAYARA /usr/local/payara5` if you are using a `csh`-like shell)
```

Reply:
This was my mistake, cutting this out. Good catch. I thought this was related to the Payara upgrade in 5.6 and wasn't needed. I'll keep it in the release notes moving forward.

Reply:
added back in https://github.com/IQSS/dataverse/commit/b629570f93460b5bba77fb201ef0713071d2d6d1